handbook/tutorial.md

```diff
@@ -1,5 +1,26 @@
 # Contributing to the Climate-Health CAFÉ Dataverse Collection
 
+## Guidelines for Dataset Contributions
+
+We strongly encourage the community of practice to contribute to the expansion of the CAFE Dataverse Collection. Emphasizing open access and collaborative research, the CAFÉ Collection invites contributions from a diverse array of stakeholders, including government agencies, NGOs, community-based organizations, industry partners, and academics. 
+
+Parameters for what datasets are appropriate and inappropriate for the CAFE Dataverse are described below:
+
+## General Guidance
+
+- Contributions should be relevant to climate and health research.
+- Contributions should not be identical to data stored in other repositories. The submission of processed derivatives or expansions of data accessible through existing sharing resources (ie: SEDAC, Google Earth Engine) are encouraged.
+- Contributions should be in line with the licensing of raw source data.
+- Data contributors should only post data that they own, have generated, or have been granted permission to reshare in a manipulated version (ie: census data).
+- No restricted access data (ie: data including personal identifying information) should be shared through the CAFE Dataverse. Contributions will be widely accessible to Harvard Dataverse users.
```

Comment:
maybe "restricted or regulated"?

Reply:
Restricted access/restricted use data has a specific definition around identifiability which I think is useful. Are there specific aspects of 'regulated' that we could add in parentheses to be clear about what we mean?